bin/nexus-blobstores-create

```diff
@@ -79,9 +79,9 @@ while [[ $# -gt 0 ]]; do
                 -d @-
             status=$?
             if [[ $status -eq 0 ]]; then
-                echo >&2 "INFO Successfully updated ${blobstore_type} blobstore: ${blobstore_name}"
+                echo >&2 " OK updated"
             else
-                echo >&2 "ERROR Failed to update ${blobstore_type} blobstore: ${blobstore_name}"
+                echo >&2 " FAIL"
```

Comment:
IUF will not be able to recognize this message if ERROR is removed. The additional information is for analysis.